strongbox-web-core/src/main/java/org/carlspring/strongbox/exception/ExceptionHandlindOutputStream.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * 
+ */
+package org.carlspring.strongbox.exception;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import org.eclipse.jetty.io.EofException;
+import org.apache.commons.io.output.ProxyOutputStream;
+
+/**
+ * @author Bogdan Sukonnov
+ *
+ */
+public class ExceptionHandlindOutputStream extends ProxyOutputStream
```

Comment:
This class' name has a typo, it should be called `ExceptionHandlingOutputStream` (`handling`, not `handlind`).

Could you please correct this?

Reply:
thanks, Martin! fixed it

Reply:
Thanks! :)